docs/static/spec/openapi/logstash-api.yaml

```
@@ -170,60 +323,210 @@ paths:
-                            peak_max_in_bytes: 35782656
-                            max_in_bytes: 35782656
-                            committed_in_bytes: 8912896
-                          old:
-                            peak_used_in_bytes: 148656848
-                            used_in_bytes: 148656848
-                            peak_max_in_bytes: 715849728
-                            max_in_bytes: 715849728
-                            committed_in_bytes: 229322752
-                          young:
-                            peak_used_in_bytes: 71630848
-                            used_in_bytes: 2740472
-                            peak_max_in_bytes: 286326784
-                            max_in_bytes: 286326784
-                            committed_in_bytes: 71630848
-                      gc:
-                        collectors:
-                          old:
-                            collection_time_in_millis: 607
-                            collection_count: 12
-                          young:
-                            collection_time_in_millis: 4904
-                            collection_count: 1033
-                      uptime_in_millis: 1809643
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: '#/components/schemas/JVMStats'
+                  - $ref: '#/components/schemas/ProcessStats'
+                  - $ref: '#/components/schemas/EventsStats'
+                  - $ref: '#/components/schemas/FlowStats'
+                  - $ref: "#/components/schemas/PipelineStats"
+                  - $ref: '#/components/schemas/QueueStats'
+                  - $ref: '#/components/schemas/ReloadStats'
+
+  /_node/stats/events:
+    get:
+      summary: Get events statistics
+      description: Return statistics related to event processing.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing events statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/EventsStats"
+
+  /_node/stats/flow:
+    get:
+      summary: Get flow statistics
+      description: Return throughput and backpressure details.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing flow statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/FlowStats"
+
+  /_node/stats/jvm:
+    get:
+      summary: Get flow statistics
```

Should this differ from the other operation? For example:
```suggestion
      summary: Get JVM statistics
```